### PR TITLE
download cards

### PR DESCRIPTION
**Pull request by harishkumar-srijan**

### CODE REVIEW

File: stories/Molecules/Blocks/DownloadCard/DownloadCard.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+.download__card {
+  &__title {
+    @extend %paragragh-font-20;
+    margin-bottom: $spacing-04;
+  }
+  &__format {
+    @extend %paragragh-font-16;
+    margin-bottom: $spacing-16;
+    color: #55606E;
```
Comment (surbhi90) [inline stories/Molecules/Blocks/DownloadCard/DownloadCard.scss:40]: @harishkumar-srijan do we have variable for this?

File: stories/Molecules/Blocks/DownloadCard/DownloadCard.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  Image: 'with-image',
+  Color: 'without-image',
+}
+
+export const DownloadCard = ({ image, image2, title, format, download, ...args }) => {
+  return (
+    <div class="download__card">
+      {image && <Publicationthumb />}
+      {image2 && <Cardthumb />}
+      <div class="download__card__description">
```
Comment (surbhi90) [inline stories/Molecules/Blocks/DownloadCard/DownloadCard.jsx:24]: @harishkumar-srijan it should be download-card as __ will be used with the element of block

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Molecules/Blocks/DownloadCard/DownloadCard.jsx (hunk 2 above)
Comment (harishkumar-srijan) [reply stories/Molecules/Blocks/DownloadCard/DownloadCard.jsx]: done

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**